repairnator/scripts/config/repairnator.cfg

```diff
@@ -15,6 +15,7 @@ NOTIFY_TO= # email adresses separated by comma (used for notifier, you must have
 NOTIFY_ENDPROCESS=0 # If set to 1, the end of dockerpool and scanner will send a notification using smtp and notify_to information
 RUN_ID_SUFFIX= # A suffix to add to the run id (useful to retrieve a specific run in serialized data).
 REPAIR_TOOLS=Nopol,AstorJMut,NPEFix # The available repair tools to use. Possible values (comma separated): Astor, Nopol, NPEFix, AssertFixer, AstorJMut, AstorJKali, AstorJGenProg
+JAVA_OPTS= # optional arguments for java such as Xmx
```

Comment:
It's more an advanced config, so it should be put at the end of the file.

Reply:
Done. It didn't seem to fit any label so I made a new one